deb-get

```diff
@@ -169,13 +169,15 @@ function get_github_releases() {
             fancy_message info "Updating ${CACHE_FILE}"
             local URL="https://api.github.com/repos/${1}/releases${2:+/$2}"
             wgetcmdarray=(wget  "${HEADERPARAM}" "${HEADERAUTH}" -q --no-use-server-timestamps "${URL}" -O- )
-            "${wgetcmdarray[@]}" | sed '/browser_download/!d;/\.deb/!d' | ${ELEVATE} tee "${CACHE_FILE}" > /dev/null || ( fancy_message warn "Updating ${CACHE_FILE} failed." )
+            "${wgetcmdarray[@]}" | ${ELEVATE} tee "${CACHE_FILE}" > /dev/null || ( fancy_message warn "Updating ${CACHE_FILE} failed." )
             if [ -f "${CACHE_FILE}" ] && grep "API rate limit exceeded" "${CACHE_FILE}"; then
                 fancy_message warn "Updating ${CACHE_FILE} exceeded GitHub API limits. Deleting it."
                 ${ELEVATE} rm "${CACHE_FILE}" 2>/dev/null
             fi
-            # we only want the .deb files download urls from the json
-            ${ELEVATE} sed -i '/browser_download/!d;/\.deb/!d' ${CACHE_FILE}
+            if [ ! ${APP} == "azuredatastudio" ]; then
```

Comment:
`if [ "${APP}" != "azuredatastudio" ]; then`
?

Reply:
They bury the URLs we need outside the browser_download parts which are all we need and cache for **every** other project.
Pretty frustrating given this app is being deprecated in favour of VSCode!